### PR TITLE
DbCollection annotation for specifying DBCollection on model class.

### PR DESCRIPTION
**Pull request by szymonsobocik**
Issue #13

### CODE REVIEW

File: src/test/java/com/futureprocessing/documentjuggler/commons/DbCollectionAnnotationTest.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        String collectionName = CollectionExtractor.getCollectionName(ModelWithDBCollectionAnnotation.class);
+
+        //then
+        assertThat(collectionName).isEqualTo(EXPECTED_COLLECTION);
+    }
+
+    @Test
+    public void shouldGetCollectionFromModel() {
+        //given
+        BDDMockito.given(mockDB.getCollection(EXPECTED_COLLECTION)).willReturn(mockDBCollection);
```
Comment (exys666) [inline src/test/java/com/futureprocessing/documentjuggler/commons/DbCollectionAnnotationTest.java]: static import

**Comment by exys666** [comment]
+1

File: src/test/java/com/futureprocessing/documentjuggler/commons/DbCollectionAnnotationTest.java (hunk 1 above)
Comment (szymonsobocik) [inline src/test/java/com/futureprocessing/documentjuggler/commons/DbCollectionAnnotationTest.java]: I wanted it do be explicit.

**Comment by pcmiel** [comment]
+1

**Pull request merged**